Utils/Dataflow/009_oracleConnector/query/datasets.sql

```diff
@@ -15,14 +17,6 @@ FROM
 WHERE
   jd.type IN ('input', 'output') AND
       t.timestamp > to_date('%s', 'dd-mm-yyyy hh24:mi:ss') AND
-      t.timestamp <= to_date('%s', 'dd-mm-yyyy hh24:mi:ss')
-  AND to_char(NVL(substr(regexp_substr(tt.jedi_task_parameters, '"taskType": "(.[^",])+'),
-                           regexp_instr(
-                               regexp_substr(tt.jedi_task_parameters, '"taskType": "(.[^",])+'),
-                               '(": ")+',
-                               1,
-                               1,
-                               1
-                           )
-                    ), '')) = 'prod'
+      t.timestamp <= to_date('%s', 'dd-mm-yyyy hh24:mi:ss') AND
+      t.pr_id > 300
 ORDER BY t.taskid;
```

Comment:
Why `t.pr_id > 300`?

Reply:
Sorry, this comment was lost. 
We substituted the condition of production task from `taskType =  'prod' ` to `t.pr_id > 300`, because it's turned out that all tasks with requestID more than 300 is production. This method is used on PanDA monitoring and discussed with Misha Borodin.